app/controllers/concerns/account_controller_concern.rb

```diff
@@ -46,4 +46,14 @@ def actor_url_link
   def webfinger_account_url
     webfinger_url(resource: @account.to_webfinger_s)
   end
+
+  def temporary_suspension_response
+    expires_in(3.minutes, public: true)
+    set_instance_presenter
+
+    respond_to do |format|
+      format.html { render 'accounts/suspended', status: 410 }
```

Comment:
I am slightly worried this could cause some implementations to entirely delete information about the account. We entertained the idea of deleting something on 410 before, and isn't it what `tootctl accounts cull` would do?

Reply:
Yeah but I think it should be fine as long as the json-ld variant returns HTTP 200...

Reply:
Speaking of which, I need to make that one use DeleteAccountService instead of SuspendAccountService

Reply:
I'm still uneasy with returning 410 here, but 200 doesn't seem super fitting either… just checked Twitter and it returns 200 for suspended accounts…

Likewise, RSS, is, I think, used by Hubzilla, and I would worry about it deciding to delete accounts on 410 (I don't know if it does, I haven't checked).